README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
 - Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
+- CCNP project collects container level primitives by implementing unified APIs defined in [CC Trusted API](https://github.com/cc-api/cc-trusted-api). The project will be moved to [CC Trusted API](https://github.com/cc-api/cc-trusted-api) in the near future. 
 
 
-## 2. Design
-
-`CCNP` includes several micro-services as BaaS(Backend as a Service) to provides
-cloud native measurement, then exposes `CC trusted API` via cloud native SDK:
-
-- Services are designed to hide the complexity of different TEE platforms and provides
-common interfaces and scalability for cloud-native environment to address the fetching
-the fetching of quote, measurement and event log.
+## 2. Installation
 
-- SDK is provided to simplify the use of the service interface for development,
-it covers communication to the service and parses the results from the services.
-With such SDK, users can perform related actions with one simple API call.
+### 2.1 Configuration
 
-- A `CCNP` device plugin is provided as the dependency for services such as Quote
-Server and Measurement Server. It will help with device mount and folder injection
-within the service.
+CCNP collects primitives of confidential cloud native environments running in confidential VMs, such as Intel® TD. You can setup an Intel® TDX enlightened host and then boot TD guest on it. The feasible configurations are as below.
 
-SDK PyPI package can be found [here](https://pypi.org/project/ccnp/). Please check our [documentation](https://intel.github.io/confidential-cloud-native-primitives/) for more details.
+|  CPU | Host OS  | Host packages  | Guest OS  | Guest packages  | DCAP packages |
+|---|---|---|---|---|---|
+|  Intel® Emerald Rapids | Ubuntu 22.04| Build packages referring to [here](https://github.com/intel/tdx-tools/tree/tdx-1.5/build/ubuntu-22.04) | Ubuntu 22.04 | Build packages referring to [here](https://github.com/intel/tdx-tools/tree/tdx-1.5/build/ubuntu-22.04) | [here](https://download.01.org/intel-sgx/sgx-dcap/1.19/linux/distro/ubuntu22.04-server/)
```

Comment:
It is a bit odd to have a column for the CPU and just repeat it twice. You can't merge the columns in markdown so I can't think of a better way to do this.   Is RHEL not supported?   Also, DCAP and attestation is one of the trickiest things to implement and understand.  At the very least I would add "Attestation" to the DCAP column. Not everyone will know that DCAP means attestation.  The two links go to very different places. The top link is going to be hard for people to follow. Linking to the linux pdf install guide and also linking to the package should be adequate. Do the docker instructions on the CCNP site for PCCS and QGS also work with the MVP stack? If so I would use the same links.  I also think a very brief explanation of what PCCS does and what the QGS is used for right before this table would help a lot.   You could say something like "The Platform certificate caching service (PCCS) is used to retrieve and cache PCK certificates locally to your cluster from Intel's Platform Certificate Service. This is necessary to attest the authenticity of a TDVM before a workload is started in it.  The Quote Generate Service (QGS) runs on the host in a specialized enclave to generate and use TD quotes. For convenient setup these can run inside a Docker container. Learn more at https://download.01.org/intel-sgx/sgx-dcap/1.17/linux/docs/Intel_TDX_DCAP_Quoting_Library_API.pdf"  I would embed this link in the text. 

The MVP related instructions will go away eventually so I don't think it is worth a lot of effort to update this section. If the docker CCNP instructions work for that section then link that instead.

Reply:
Mid-stream distro also works for CentOS Stream 9, but it will not be put here until CCNP is validated with it.
Containerized PCCS and QGS work for both rows. However, it's just an alternative for mid-stream because they may release PCCS and QGS packages for Ubuntu 23.10 in future. The configuration table will be evolved along with mid-stream status.

Reply:
If the same Docker instructions work for both TDX integrations then I would just use that. This table will evolve as distros update their TDX offerings.